theurgy.lic

```diff
@@ -459,22 +459,21 @@ class TheurgyActions
 
   def commune_eluned
     safe_walk_to @data['dirt_foraging']['id']
-    forage? 'dirt'
-
-    get_from_container(@water_holder)
-    match = bput('commune eluned',
-                 'completed this commune too recently',
-                 'You struggle to commune',
-                 'you have attempted a commune too recently in the past',
-                 'You grind some dirt in your fist')
-    put_in_container(@water_holder)
-    if [left_hand, right_hand].include?('dirt')
-      bput('drop dirt', 'You drop some',
-           'But you aren\'t holding',
-           'What were you referring')
+    if forage?('dirt', 5
+      get_from_container(@water_holder)
+      match = bput('commune eluned',
+                   'completed this commune too recently',
+                   'You struggle to commune',
+                   'you have attempted a commune too recently in the past',
+                   'You grind some dirt in your fist')
+      put_in_container(@water_holder)
+      if [left_hand, right_hand].include?('dirt')
+        bput('drop dirt', 'You drop some',
+             'But you aren\'t holding',
+             'What were you referring')
+      end
+      match == 'You grind some dirt in your fist'
     end
-
-    match == 'You grind some dirt in your fist'
```

Comment:
With this conditional no longer outside the `if` block, I'd like to see an explicit return. What do you think?

Reply:
TBH I have no idea what this LOC is for, afaict it evaluates to true and moves on.

Reply:
This is an implicit return. Ruby functions return the value of the last evaluated expression.

Reply:
Done. Maybe that will solve the issue with the commune being retried. If not, it will at least provide a false to base the control flow off of. Thanks.